docs/source/alfacase_definitions/LinearIPRDescription.txt

```diff
@@ -17,6 +19,14 @@
             min_pressure_difference: 
                 value: number
                 unit: string # optional
+            well_index_input_type: \ :class:`MultiInputType <alfasim_sdk._internal.constants.MultiInputType>`\ # optional
             well_index: 
                 value: number
                 unit: string # optional
+            well_index_curve: 
+                image:
+                    values: [number]
+                    unit: string
+                domain:
+                    values: [number]
+                    unit: string # optional
```

Comment:
`well_index_curve` is optional, not  `well_index_curve.domain.unit` another open PR already fix this,